.github/pull_request_template.md

```diff
@@ -1,13 +1,20 @@
+<!-- Thank you for offering your contribution with Deals for Devs! -->
+
 ## Description
-<!-- Describe your contribution and its benefits in a sentence  -->
+<!-- Briefly explain what this PRs purpose is. -->
 
-## Related Ticket(s) and Documentation(s)
-<!-- Use `Closes #issue number` if your PR fixes an issue or `Relates #issue number` if it connects to the issue number -->
+## Issue
+<!-- If this PR is related to a specific issue # please list that here. (e.g - "Closes: XXXX") -->
 
-## Screenshots
+## Aditional Information
+<!-- This can be any comments you feel are necessary to share or even images that may be useful. -->
 
-<!-- Add all the screenshots which support your changes -->
+## PR Requirements
+<!-- Within the [ ] remove the space with an x -->
+- [ ] I have carefully read through and understand the [Deals-for-Devs Contributing Guide](https://github.com/Learn-Build-Teach/deals-for-devs/blob/dev/.github/contributing.md)
+- [ ] The title of this PR follows the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) format
+- [ ] The `Description` gives a good representation of the changes made
+- [ ] If this PR addresses an open Issue, it is linked in the `Issue` section
 
-## Note to reviewers
 
-<!-- Add notes to reviewers if applicable -->
+<!-- Once again we thank you for your time and effort in helping us alongside this open source project! -->
```

Comment:
[suggestion] This section is where contributors add their thoughts, so it'd be best to put this information instead of a thank you note. 
```suggestion
<!-- If you have any additional thoughts, just add them here. -->
```

Reply:
Updated

- Completed